src/app/core/topbar/components/topbar.component.html

```diff
@@ -29,7 +29,11 @@
         <button class="btn btn-outline-secondary" ngbDropdownToggle>
           <span class="hidden-lg-down"> {{ user.displayName }}</span>
         </button>
-        <div ngbDropdownMenu class="dropdown-menu dropdown-menu-right">
+        <div
```

Comment:
I'm not used anymore to the angular code, but does this mean that when `platformIsLive` is set to true the logout is shown? The behavior should be when `platformIsLive` is true, the logout should be hidden.

Reply:
@LucWollants true, there should be an `!` in front of it.